.env.local.example

```diff
@@ -0,0 +1,9 @@
+# Algolia
+NEXT_PUBLIC_ALGOLIA_APP_ID=
+NEXT_PUBLIC_ALGOLIA_SEARCH_API_KEY=
+NEXT_PUBLIC_ALGOLIA_BASE_SEARCH_INDEX_NAME=
+
+# GitHub API
+# Note: this token expires on Dec 16, 2023
+# check fine-grained tokens https://docs.github.com/en/authentication/keeping-your-account-and-data-secure/creating-a-personal-access-token#about-personal-access-tokens
```

Comment:
I don't think we necessarily need to use the fine-grained tokens here... It's a public repo, so a simple classic read-only key should do.
<img width="432" alt="image" src="https://user-images.githubusercontent.com/54227730/208180297-a8b9669b-0cc7-4042-8ca4-3d9cae396f15.png">
<img width="667" alt="image" src="https://user-images.githubusercontent.com/54227730/208181106-64be6b08-b5a7-4e35-9761-a15318cf1240.png">
Just give it a name and set an expiration, and don't check any boxes, then save.

When I try to add a fine-grained token and select the geth-website repo (or the go-ethereum repo) I am unable to do so, so I'm not sure if such a token would really add any benefit.

```suggestion
# About personal access tokens https://docs.github.com/en/authentication/keeping-your-account-and-data-secure/creating-a-personal-access-token#about-personal-access-tokens
```

Reply:
I prefer to use more restricted ones by default, fine-grained are more secure than classic tokens.

Reply:
Right, totally agree normally, but personally I am not even able to restrict a fine-grained token to just this repo... and read-only of public data is about as strict as it gets. 
I guess I'm suggesting this for two reasons, 1- the fine-grained tokens are still in beta, and 2- I just don't want someone to try and generate one and it not work and just want to keep it as simple as possible

Looking at this, I'm pretty sure for what we need this for they're essentially the same. So ultimately not a big deal either way, fine with leaving it